### PR TITLE
CWD in Launcher

### PR DESCRIPTION
**Pull request by roshancvp**
Screenshot:
screen shot 2016-07-20 at 11 40 07 am

### CODE REVIEW

**Comment by jasongrout** [comment]
cf. https://github.com/jupyter/jupyterlab/issues/316

**Comment by jasongrout** [comment]
Can you double-check that the terminal and console are started with the correct cwd?

**Comment by roshancvp** [comment]
@jasongrout Not yet, the console and terminal are started in the home directory instead of the current path of the file browser.

**Comment by jasongrout** [comment]
cf. #483 about a discussion of the terminal/console opening in the cwd.

**Comment by jasongrout** [comment]
CC @ellisonbg - you asked for this...what do you think?

**Comment by faricacarroll** [comment]
The launcher got a little crowded with all the new information, so we updated the style.

We decided to get rid of "This is not ready for general usage yet." and replace it with a cheeky message that communicates how buggy Jupyterlab is (Brian gave us the go-ahead for the humor). It generates a slightly different phrase each time the page is launched. 

It rotates through the following phrases: 

- super alpha preview
- very alpha preview
- extremely alpha preview
- exceedingly alpha preview
- alpha alpha preview
- it’s buggy!

If you have any suggestions for more phrases, please let us know!

screen shot 2016-07-21 at 3 06 47 pm

@spoorthyv @roshancvp

**Comment by jasongrout** [comment]
Can we delete the "It's buggy!"? (1) I think we don't want to promote an explicit association of Jupyter with bugginess, and (2) it doesn't really convey the idea that this is a preview of things to come, we're just not ready yet.

**Comment by jasongrout** [comment]
+1 for making it not so crowded

File: src/landing/plugin.ts [hunk 1]
```diff
@@ -43,8 +43,8 @@ function activateLanding(app: Application, services: ServiceManager, pathTracker
   logo.className = 'jp-ImageJupyterLab jp-Landing-logo';
   dialog.appendChild(logo);
 
-  let previewMessages = ["super alpha preview", "very alpha preview", "extremely alpha preview", "exceedingly alpha preview", "alpha alpha preview", "it’s buggy!"];
-  let ranNum = Math.floor(Math.random()*6);
+  let previewMessages = ["super alpha preview", "very alpha preview", "extremely alpha preview", "exceedingly alpha preview", "alpha alpha preview"];
+  let ranNum = Math.floor(Math.random()*5);
```
Comment (jasongrout) [inline src/landing/plugin.ts]: `Math.floor(Math.random()*previewMessages.length)`

File: src/landing/plugin.ts [hunk 2]
```diff
@@ -109,6 +102,33 @@ function activateLanding(app: Application, services: ServiceManager): void {
     app.commands.execute('terminal:create-new');
   });
 
+  let cwd = document.createElement('div');
+  cwd.className = 'jp-Landing-cwd';
+
+
+  let folderImage = document.createElement('span');
+  folderImage.className = 'jp-Landing-folder';
+  folderImage.style.height = "12px";
```
Comment (blink1073) [inline src/landing/plugin.ts]: These should be in the style sheet instead of inlined.

File: src/landing/plugin.ts [hunk 3]
```diff
@@ -109,6 +102,33 @@ function activateLanding(app: Application, services: ServiceManager): void {
...
+  folderImage.style.width = "12px";
```
Comment (blink1073) [inline src/landing/plugin.ts]: You added these to the style sheet but forgot to remove them from the code ;)

Comment (faricacarroll) [inline src/landing/plugin.ts]: 😅

**Comment by blink1073** [comment]
Well done, thanks!

**Pull request merged**